finder-app/doc.txt

```diff
@@ -0,0 +1 @@
+Goofbye
```

Comment:
@Sharath-Jonnala consider whether you need/want to check these two files into your repo.